torchao/csrc/cuda/fp6_llm/ptx_mma.cuh

```diff
@@ -43,9 +46,9 @@
 // MODIFICATION NOTE: to support MSVC
 // - uint32_t __restrict__ Reg[][4] is changed to uint32_t (* __restrict__ Reg)[4]
 // - half __restrict__ (*read_SPTR) is changed to half (* __restrict__ read_SPTR)
-template <typename TilingConfig>
+template <typename T, typename TilingConfig>
 __device__ __forceinline__ void B_FromSharedToReg(uint32_t (* __restrict__ Reg)[4],
```

Comment:
Same as another comment. Do we need to care if T is half or bfloat16? (but this time the input argument is typed so maybe yes)